orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/CreateServerGroupTask.groovy

```diff
@@ -48,7 +49,10 @@ class CreateServerGroupTask extends AbstractCloudProviderAwareTask implements Re
     if (!creator) {
       throw new IllegalStateException("ServerGroupCreator not found for cloudProvider $cloudProvider")
     }
-
+    def validationError = new ClusterNameValidator().validateClusterName(stage, cloudProvider)
+    if (validationError) {
+      throw new IllegalStateException("Cluster name validation failed with message : $validationError")
```

Comment:
I think this concatenated error message is going to be quite verbose

```
"Cluster name validation failed with message: Invalid cluster name 'blah' Must be less than 10 characters for aws"
```

Reply:
thanks, fixed it